README.md

```diff
@@ -16,6 +16,14 @@ function lovr.update(dt)
 end
 ```
 
+lovr-mouse can only run on systems where LÖVR uses both LuaJIT and GLFW. If your project needs to be compatible with Android or WebVR, you can optionally include lovr-keyboard with:
+
+```lua
+if type(jit) == 'table' and lovr.getOS() ~= 'Android' and lovr.getOS() ~= 'Web' then
+	lovr.keyboard = require 'lovr-mouse'
```

Comment:
There's just a small typo here and in the sentence above (lovr.keyboard instead of lovr.mouse).  It would also be good if this line was indented with spaces to match the example code above.